aether-utils/aether-client/aether/client/test_fixtures/__init__.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2018 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from aether.client import fixtures as fix
+import os
+import pytest
+
+from aether.client import Client
+
+URL = os.environ['KERNEL_URL']
+USER = os.environ['KERNEL_ADMIN_USERNAME']
+PW = os.environ['KERNEL_ADMIN_PASSWORD']
```

Comment:
I would suggest to rename this variable names `KERNEL_USERNAME`  and `KERNEL_PASSWORD`; including `ADMIN` in the name implies that non admin users cannot work with the client :thinking: 

Another point, the SQL readonly user... is still a thing with this new client? :thinking:

Reply:
SQLReadonly if i remember is only for direcly accessing Postgres (like the consumer does) not using the kernel api.

Reply:
That's not entirely true, in kernel it's `ADMIN_XXX` (no need of `KERNEL_` prefix), bit in the other modules it's `AETHER_KERNEL_XXX`

This is what I mean.

```python
URL = os.environ['AETHER_KERNEL_URL']
USER = os.environ['AETHER_KERNEL_USERNAME']
PW = os.environ['AETHER_KERNEL_PASSWORD']
```

```yml
   # ---------------------------------
   # Aether Kernel Client
   # ---------------------------------
   client-test:
    build: ./aether-utils/aether-client
    environment:
      AETHER_KERNEL_URL: http://kernel-test:9000
      # In the container      # In the `.env` file
      AETHER_KERNEL_USERNAME: ${KERNEL_ADMIN_USERNAME}
      AETHER_KERNEL_PASSWORD: ${KERNEL_ADMIN_PASSWORD}
    command: test
```

Reply:
Ah, :+1:

Reply:
Sorry, sometimes it's hard to me to explain even the most simple idea... :grimacing: